### ISSUE TITLE
Compiled image

### ISSUE THREAD

**Issue by nikolas-n** [atopuzov/c201#2]
Could you upload an compiled image of your installation, so that we can dd it in an SD card?

Or would it be possible to have the compiled kernel you're using? I've installed devsus and was thinking to change the kernel so that the wifi works as well as the hdmi?

**Comment by atopuzov** [comment]
Hi,

I'm primarily using Arch Linux now [1]. Which uses 3.14 kernel (the google one). All 4.x kernels are very buggy for me (sd card driver) and sometimes don't boot. Seems to be the most stable solution. I don't use 3D acceleration (OpenGL, armsoc driver) i use fbturbo xf86 driver. Haven't really tested HDMI well with fbturbo but with fbdev xf86 driver works very well.
You can try using devuan [2] (go for chromeveyron image) and switch to debian. 

[1] https://archlinuxarm.org/platforms/armv7/rockchip/asus-chromebook-flip-c100p
[2] https://mirror.leaseweb.com/devuan/devuan_jessie/embedded/

**Comment by nikolas-n** [comment]
In fact, I'm already using Devuan after having tried Arch Linux (which I installed following the first link you cited). I'm not really confident in switching to Debian from Devuan. Can I leave the kernel intact or should I use another one?

Also, do you happen to know if and how I can make the proprietary wifi work under Devuan? That'd be very useful.

**Comment by atopuzov** [comment]
If I remeber corectly I just updated my apt sources and did a `apt-get update && apt-get dist-upgrade`.
I didn't use wifi as it wasn't working, used a USB ethernet adapter. Might still have the it installed on one of the USB sticks so I can check. 
Did you try using the files from 1 and the brcmfmac driver? I see that [2] actually has that.

[1] https://chromium.googlesource.com/chromiumos/third_party/linux-firmware/+/master/brcm/
[2] https://github.com/parazyd/arm-sdk/tree/master/extra/chromeveyron/lib/firmware

**Comment by nikolas-n** [comment]
I tried to, but I'm not sure if I'm doing it correctly. Shouldn't the drivers be loaded by the kernel? How can I load them after I've booted into Devuan?

> Might still have the it installed on one of the USB sticks so I can check.

That would help a lot! 😃

**Comment by nikolas-n** [comment]
Hi,

Is there any update? I'm still struggling...

**Comment by hyshka** [comment]
@nikolas-n as with @atopuzov I have found the most success with the ArchLinuxArm 3.14 kernel, following these instructions to install: https://archlinuxarm.org/platforms/armv7/rockchip/asus-chromebook-flip-c100p

I can't provide help with migrating from Devuan to Debian but I'd recommend a fresh install of Arch. I haven't gotten the OpenGL/armsoc stuff working either but HDMI did work on my initial install...and then I broke it by installing the fbturbo driver.

**Comment by nikolas-n** [comment]
@hyshka , you're totally right. That's what I did like two months ago (almost some days after that post) and I've been enjoying Arch on my machine since.

**Comment by atopuzov** [comment]
I have recently picked up my old chromebook again and started playing around.
linux-c201 package [1] works very well (seems crypto is broken as I have issues with ecryptfs)
I also managed to get a working u-boot by compiling [2], packages are here [3] (make sure your rootfs partition is marked as bootable as uboot looks for partitions marked bootable, for fdisk it is in advanced settings, make partition legacy bootable).

[1] https://github.com/urjaman/arch-c201 && https://urja.dev/linux-c201/ (if using u-boot don't use the -chromebook package)
[2] https://github.com/archlinuxarm/PKGBUILDs/pull/1618
[3] https://github.com/atopuzov/c201/releases/tag/20190513
